backend/api-design.md

```diff
@@ -11,12 +11,12 @@
 - [`/tournaments/<id>/actions/close-registrations` (POST)](#tournamentsidactionsclose-registrations)
 - [`/tournaments/<id>/actions/open-registrations` (POST)](#tournamentsidactionsopen-registrations)
 - [`/tournaments/<id>/actions/trigger-matchmaking` (POST)](#tournamentsidactionstrigger-matchmaking)
-- [`/tournaments/<id>/players` (POST)](#tournamentsidplayer)
+- [`/tournaments/<id>/players` (POST, GET)](#tournamentsidplayer)
 - [`/tournaments/<id>/players/<id>` (PUT, DELETE, GET)](#tournamentsidplayerid)
-- [`/tournaments/<id>/registrations` (POST)](#/tournamentsidregistrations)
-- [`/tournaments/<id>/registrations/<id>` (PUT, DELETE, GET)](#/tournamentsidregistrationsid)
-- [`/tournaments/<id>/matches` (POST)](#tournamentsidmatches)
-- [`/tournaments/<id>/matches/<id>` (PUT, DELETE, GET)](#tournamentsidmatches)
+- [`/tournaments/<id>/registrations` (POST, GET)](#tournamentsidregistrations)
+- [`/tournaments/<id>/registrations/<id>` (PUT, DELETE, GET)](#tournamentsidregistrationsid)
+- [`/tournaments/<id>/matches` (POST, GET)](#tournamentsidmatches)
+- [`/tournaments/<id>/matches/<id>` (PUT, GET)](#tournamentsidmatches)
```

Comment:
Please also add a disclaimer up here that pagination is out of scope for the first iteration of the API.

I can see that this will be a first feedback comment when others read sentences like "will return all players in the tournament".